client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -430,27 +430,35 @@ class PluginMeta extends Component {
 	}
 
 	render() {
+		const {
+			selectedSite,
+			plugin: pluginProp,
+			isPlaceholder,
+			sites,
+			atEnabled,
+			slug,
+			translate,
+			isMock,
+		} = this.props;
 		const cardClasses = classNames( 'plugin-meta__information', {
 			'has-button': this.hasOrgInstallButton(),
-			'has-site': !! this.props.selectedSite,
-			'is-placeholder': !! this.props.isPlaceholder
+			'has-site': !! selectedSite,
+			'is-placeholder': !! isPlaceholder
 		} );
-
-		const plugin = this.props.selectedSite && this.props.sites[ 0 ] && this.props.sites[ 0 ].plugin
-			? this.props.sites[ 0 ].plugin
-			: this.props.plugin;
-		const path = ( ! this.props.selectedSite || plugin.active ) && this.getExtensionSettingsPath( plugin );
+		const selectedSitePlugin = get( sites, '[0].plugin', pluginProp );
+		const plugin = selectedSite ? selectedSitePlugin : pluginProp;
```

Comment:
Oh, wait – I see what's happening. I was confused by the default value of the `get` call and the variable being called `selectedSitePlugin` (because it might not be, it might be the dotorg plugin). If we stick with this logic, can the variables be better named to avoid confusion?